core/src/main/java/apoc/ApocConfig.java

```diff
@@ -96,7 +96,8 @@ public enum UuidFormatType { hex, base64 }
     public static final String APOC_CONFIG_JOBS_QUEUE_SIZE = "apoc.jobs.queue.size";
     public static final String APOC_CONFIG_INITIALIZER = "apoc.initializer";
     public static final String LOAD_FROM_FILE_ERROR = "Import from files not enabled, please set apoc.import.file.enabled=true in your apoc.conf";
-
+    public static final String APOC_MAX_DECOMPRESSION_RATIO = "apoc.max.decompression.ratio";
+    public static final Integer DEFAULT_DECOMPRESSION_RATIO = 100;
     /**
```

Comment:
I think with apoc config there's currently no way to write constraints like min > 1. With the neo4j config there is. Also I don't know whether we want to allow negative numbers (=no protection against zip bombs). What do you think?

Reply:
For negative number question I would ask Alex in security team. From a technical standpoint I think either is fine. If we end up not allowing it, I guess we can have a check and fail when it is used, even if that is not as elegant as the Neo4j way.